api/src/main/java/com/datastrato/gravitino/rel/expressions/literals/Literals.java

```diff
@@ -98,6 +99,17 @@ public static LiteralImpl<Double> doubleLiteral(Double value) {
     return of(value, Types.DoubleType.get());
   }
 
+  /**
+   * Creates a decimal type literal with the given value.
+   *
+   * @param value the decimal literal value
+   * @return a new {@link Literal} instance
+   */
+  public static LiteralImpl<Decimal> decimal(Decimal value) {
```

Comment:
Is it better renaming to `decimalLiteral`?

Reply:
At first, I intended to use the type name directly as the method name, such as `long()`, `float()`, `double()`. However, as they are reserved keywords in Java, I had to resort to using `longLiteral()`, `floatLiteral()`, `doubleLiteral()`. 

If we want to use `decimalLiteral()` here, should the already defined methods like `time()`, `date()`, `string()` be changed to `xxxLiteral()`?

Reply:
Already rename to `xxxLiteral()` for maintaining a consistent naming style